ipn/ipnstate/ipnstate.go

```diff
@@ -18,13 +18,15 @@ import (
 	"sync"
 	"time"
 
+	"inet.af/netaddr"
 	"tailscale.com/tailcfg"
 	"tailscale.com/types/key"
 )
 
 // Status represents the entire state of the IPN network.
 type Status struct {
 	BackendState string
+	IPs          []string // Tailscale IP(s) assigned to this node
```

Comment:
I guess string for JSON? Does netaddr.IP not JSON-ify well?

Reply:
Not sure what I was thinking. netaddr.IP stringifies fine. Fixed.